### DISCUSSION TITLE
ilogtail 1.7.0 windows编译失败

### DISCUSSION THREAD

**Discussion by 7y-9** [alibaba/ilogtail#1209] [Help] [answered]
我尝试使用文档中提供的办法（https://ilogtail.gitbook.io/ilogtail-docs/v/1.7.0/installation/sources/build）
进行windows编译，但是编译失败。

build.stdout如下
[build.stdout.LOG](https://github.com/alibaba/ilogtail/files/13213235/build.stdout.LOG)

![image](https://github.com/alibaba/ilogtail/assets/121850020/3e5ca239-ee53-4a92-93d6-14fb5cbeb6d5)

<img width="977" alt="image" src="https://github.com/alibaba/ilogtail/assets/121850020/9a27697d-bf1d-4da4-bf47-7aa6fc48c9c6">

**Comment by 7y-9** [comment]
已经解决，此dicussion关闭

**Reply by messixukejia** [reply]
可以把解决思路贴一下

**Reply by 7y-9** [reply, marked as answer]
由于不常使用windows环境，不了解空格会导致异常问题。
windows编译需确保依赖目录中不含空格。

**Reply by pigwantacat** [reply]
老哥能写下解决步骤吗，我也遇到了这个问题
